README.md

```diff
@@ -81,7 +81,8 @@ discard: [ordered array card ids]
 - - `get api/v1/game_state/1` - Returns the game state for game ID of 1.
 - Example Response:
 - - `Status: 200`
--  ~~~~
+-  
+~~~~
```

Comment:
@DavisC0801 Can you change these characters to the ``` to follow the formatting convention we've been using in the rest of the documentation?

Reply:
Done, issue was with bullets and not the codeblock tag.